SuperBuild/python_patch.cmake

```diff
@@ -4,7 +4,7 @@ set(out_msvc9compiler ${PYTHON_SRC_DIR}/Lib/distutils/msvc9compiler.py)
 
 #
 # This custom version of 'msvc9compiler.py' has been made by applying
-# the following patches to the version of 'msvc9compiler.py' shipped with Python-2.7.3.tgz:
+# the following patches to the version of 'msvc9compiler.py' shipped with Python-2.7.10.tgz:
```

Comment:
That's correct:

    $ md5sum ./Python-2.7.3/Lib/distutils/msvc9compiler.py ./Python-2.7.10/Lib/distutils/msvc9compiler.py
    908f2eefe08c08c33a9e97558c5c2d93 *./Python-2.7.3/Lib/distutils/msvc9compiler.py
    908f2eefe08c08c33a9e97558c5c2d93 *./Python-2.7.10/Lib/distutils/msvc9compiler.py